### DISCUSSION TITLE
Cisco SecureX integration?

### DISCUSSION THREAD

**Discussion by B3DTech** [Security-Onion-Solutions/securityonion#4380] [Ideas]
Wondering if anyone has thought about creating an integration with Cisco SecureX?  We’re looking at implementing it since it’s free with our Cisco products. Being able to have SO data in it would be awesome!  Not sure what data can be integrated yet. 

I do see AlienVault OTX does have an integration. 

Would be a good bonus for users/customers.

**Comment by weslambert** [comment]
I have not tried integrating with SecureX specifically, but I'll give it a shot when I get a chance and get back with you. Or if you have a specific use case you would like to test, please let me know.
